dataprofiler/profilers/profile_builder.py

```diff
@@ -521,6 +522,16 @@ def _max_col_samples_used(self):
             samples_used = max(samples_used, col.sample_size)
         return samples_used
 
+    @property
+    def _min_col_samples_used(self):
+        """
+        Calculates and returns the number of rows that were completely sampled
+        i.e. every column in the Profile was read up to this row (possibly
+        further in some cols)
+        """
+        return min([self._profile[col].sample_size
+                    for col in self._profile], default=0)
```

Comment:
Not asking for change in this PR, but i do find it less clear as opposed to:
```python
if not self._profiles:
    return 0
return min([profile.sample_size for profile 
            in self._profile.values()])
```
I find it more explicit about saying it defaults to 0 when there are no profiles